public/app/features/alerting/unified/components/contact-points/__mocks__/grafanaManagedServer.ts

```diff
@@ -0,0 +1,24 @@
+import { rest } from 'msw';
+
+import { AlertManagerCortexConfig } from 'app/plugins/datasource/alertmanager/types';
+import { ReceiversStateDTO } from 'app/types';
+
+import { setupMswServer } from '../../../mockApi';
+
+import alertmanagerMock from './alertmanager.config.mock.json';
+import receiversMock from './receivers.mock.json';
+
+export default () => {
+  const server = setupMswServer();
+
+  server.use(
+    // this endpoint is a grafana built-in alertmanager
+    rest.get('/api/alertmanager/grafana/config/api/v1/alerts', (_req, res, ctx) =>
```

Comment:
should we move these endpoints to a `const` that we can export in case it's needed in other places?